app/build.gradle

```
@@ -55,4 +55,10 @@ dependencies {
 
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+
+    // ViewModel
+    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:"
```

Отсутствует версия в импорте. 
```suggestion
    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:2.2.0"
```

---

Я же написал комментарий. Почему не исправляешь?